### PR TITLE
feat(Attractor): initial Attractor component, update demo, update docs

### PR DESCRIPTION
**Pull request by AlexWarnes**
Hi there! Submitting this as a starting point and happy to make edits based on feedback, or if there's too much feedback feel free to hack away at it or start from scratch. Looking forward to seeing how the library grows!

**ISSUES**
1. I don't know how to allow the `<Attractor />` component to inherit its position. For example, if I want the attractors to interact with each other, I would attempt to wrap them in `<RigidBody />` tags, but as it stands I can't get that to work.
2. I'm not strong with React or threejs patterns, so there might be some obvious issues to ya'll that I've just simply overlooked (or I got it to work without knowing why).

### CODE REVIEW

File: packages/react-three-rapier/src/Attractor.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+export const Attractor = ({
+  position = [0, 0, 0],
+  strength = 0.5,
+  range = 15,
+  showHelper = false,
+  gravityType = 'static',
+  gravitationalConstant = 6.673e-11,
+}: AttractorProps) => {
+  const { world } = useRapier()
+  const gravitySource = new THREE.Vector3(position[0], position[1], position[2])
```
Comment (hmans) [inline packages/react-three-rapier/src/Attractor.tsx:24]: It's just a smaller point, but all these constants including the function below will be recreated every time this component re-renders. This should ideally not happen a lot _anyway_, but no reason to be wasteful.

**Review by hmans — CHANGES_REQUESTED** [review]

**Comment by wiledal** [comment]
> Had a couple of comments. Let me know if you need further clarification. I think it'd be a cool feature to have, but it would be nice if it could be integrated into the library a little more idiomatically.

I agree with these comments. I really like this idea, and it can easily be more integrated into the core of this lib if it's going to be part of this repo. 

- `Attractor` can be represented by an `Object3d` to allow inherited positioning in world-space
- Helper should be part of `Debug`

Would a `Repeller` just be an `Attractor` with negative force?

**Comment by AlexWarnes** [comment]
Thank you for the review and feedback! I'll work on incorporating these changes.

> Would a Repeller just be an Attractor with negative strength?

Yeah! The impulse calculation works out conveniently that you can just pass a negative strength to repel.

**Comment by hmans** [comment]
Hi! I often look at these things through the glasses of someone who's building _games_ with the stack, so a lot of the feedback I have comes from that position. And here's something I'd like to add in that regard:

Please make Attractors take an `InteractionGroup` type prop that configures 1) which interaction group it is in and 2) which interaction groups it should impact. In order to be useful in games, it needs to be possible to steer which bodies are affected by attractors, and which bodies will just ignore them.

Thanks, and let me know if you have questions about this!

**Comment by wiledal** [comment]
Updated to be more integrated as a core feature in #157

**Pull request closed without merging**